saleor/product/migrations/0046_product_category.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-01-16 11:56
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def assign_category_to_products(apps, schema_editor):
+    Product = apps.get_model('product', 'Product')
+    Category = apps.get_model('product', 'Category')
+    default_category = Category.objects.first()
+    for product in Product.objects.all():
+        product.category = product.categories.first() or default_category
```

Comment:
Why is there a default category?

Reply:
Theoretically, database allows creating products without any category. Although, in the interface we do not allow such an action so I can skip it.

Reply:
I just think that this constitutes a change to the business logic unrelated to the goal of this PR. We can certainly consider making the category field required but I think it deserves its own ticket where we could discuss the pros and cons.